functional-tests/src/test/groovy/spec/Tailings.groovy

```diff
@@ -29,14 +29,5 @@ class Tailings extends GebReportingSpec {
         then: "User can now see the tailings tab"
         assert tailingsTab.tabSelect.displayed == true
     }
-
-    def "Scenario: User navigates to the tailings tab and the required reports are visible"(){
-        when: "User clicks on the tailings tab"
-        tailingsTab.tabSelect.click()
-
-        then: "the reports are visible"
-            assert tailingsTab.document0Name != null
-
-    }
```

Comment:
@JustinMacaulay I removed this scenario to test of data is being displayed correctly. 

As per our test discussion, unit tests will be covering if data is rendered correctly using snapshots. This specific example was very fragile.

Happy to chat about this further if need be!